### ISSUE TITLE
Errors within 'start.ts' command are hidden

### ISSUE THREAD

**Issue by rhyswilliamsza** [n8n-io/n8n#5360]
**Describe the bug**
An error occurring within `start.ts`, specifically within the main `try` block of `async run()`, will be hidden from the end-user (though still reported) since 0.211.0 (introduced with https://github.com/n8n-io/n8n/pull/5127). I feel that it would be good to log out the error before reporting to Sentry, else it's pretty difficult to know what actually went wrong when things break.

**To Reproduce**

1. Using the official `n8n` docker image, configure an invalid property (i.e. port of 271867128).
2. The process will die without any warning.

**Expected behavior**
A clear error should be reported to the user before reporting & `exit(1)`.

**Environment (please complete the following information):**

- OS: All
- n8n Version: 0.211.0 onwards.

**Comment by netroy** [comment]
Thanks for reporting this. This has been addressed in https://github.com/n8n-io/n8n/pull/5342 and released in `0.214.0`.

**Comment by rhyswilliamsza** [comment]
Thanks for the reply! However I still seem to be experiencing the same side effect with `0.214.0`. Is it possible that there is an edge case where the source error is of an unexpected format (another object entirely), thus not getting [wrapped](https://github.com/netroy/n8n/blob/0ffbfc4c15e27709497e30fa82fc94382b82cd83/packages/workflow/src/ErrorReporterProxy.ts#L33) correctly to trigger the logger call? 

```
<redacted>@<redacted>:/home/<redacted># docker run -e N8N_PORT=test n8nio/n8n:0.214.0-debian
Unable to find image 'n8nio/n8n:0.214.0-debian' locally
0.214.0-debian: Pulling from n8nio/n8n
15639959ffec: Already exists                                                                                                                                                                                                                                                                                              
81ef007f1599: Already exists                                                                                                                                                                                                                                                                                              
58deacb177a2: Already exists                                                                                                                                                                                                                                                                                              
26774fe15380: Already exists                                                                                                                                                                                                                                                                                              
a48fa3230958: Already exists                                                                                                                                                                                                                                                                                              
35a0ac044756: Already exists                                                                                                                                                                                                                                                                                              
42b1188b7512: Already exists                                                                                                                                                                                                                                                                                              
eb9b0e6a0feb: Already exists                                                                                                                                                                                                                                                                                              
db68b7a5efa3: Already exists                                                                                                                                                                                                                                                                                              
4f4fb700ef54: Already exists                                                                                                                                                                                                                                                                                              
cca7d3522bb2: Pull complete
60808afe8e4e: Pull complete
df3b5ca024e4: Pull complete
1f1f3651fc52: Pull complete
Digest: sha256:7991cafbf61cb7ef59d054bb40071da89dcdcfe05e49242677d0bcfae781a67d
Status: Downloaded newer image for n8nio/n8n:0.214.0-debian
Initializing n8n process
Migrations in progress, please do NOT stop the process.
UserSettings were generated and saved to: /home/node/.n8n/config
Migrations finished.

> exit with code 1
```

**Comment by Joffcom** [comment]
Afternoon,

Looks like I forgot to update this one, This shuld have been resolved in 1.2.0. If you are still seeing this let me know and we can open it again.
